doc/reference_coverage.xml

```diff
@@ -0,0 +1,195 @@
+    </refnamediv>
+
+    <refsynopsisdiv>
+      <funcsynopsis>
+        <funcprototype>
+          <funcdef>geometry <function>ST_CoverageInvalidLocations</function></funcdef>
+          <paramdef><type>geometry winset </type>
+            <parameter>geom</parameter></paramdef>
+          <paramdef><type>float8 </type>
+            <parameter>tolerance</parameter></paramdef>
+        </funcprototype>
+      </funcsynopsis>
+    </refsynopsisdiv>
+
+    <refsection>
+      <title>Description</title>
+
+      <para>Collects all the polygons in the window partition and then checks whether they form a "clean coverage", in which no polygons have any overlap, all vertices on shared edges are identical, and no gaps between polygons exist smaller than the tolerance value.</para>
+      <para>If polygons are found that are not clean relative to their neighbors, the problematic edges will be returned as MULTILINESTRING. For clean polygons the return value will be a NULL. Any non-polygonal or empty geometries on the input will be generate NULL returns.</para>
+      <para>Availability: 3.4.0 - requires GEOS &gt;= 3.12.0</para>
+    </refsection>
+
+    <refsection>
+      <title>Examples</title>
+
+      <programlisting>CREATE TABLE coverage (id integer, geom geometry);
+
+INSERT INTO coverage VALUES
+  (1, 'POLYGON((0 0, 10 0, 10.1 5, 10 10, 0 10, 0 0))'),
+  (2, 'POLYGON((10 0, 20 0, 20 10, 10 10, 10.05 5, 10 0))'),
+  (3, 'POLYGON((20 0, 30 0, 30 10, 20 10, 20 0))');
+
+SELECT id, ST_AsText(ST_CoverageInvalidLocations(geom) OVER ())
+  FROM coverage;
+
+ id |           st_astext
+----+--------------------------------
+  1 | LINESTRING(10 0,10.1 5,10 10)
+  2 | LINESTRING(10 10,10.05 5,10 0)
+  3 |
+      </programlisting>
+
+      <programlisting>-- Test entire table for validity
+SELECT true = ALL (
+    SELECT ST_CoverageInvalidLocations(geom) OVER () IS NULL
+    FROM coverage
+    );
+      </programlisting>
+    </refsection>
+
+    <refsection>
+      <title>See Also</title>
+      <para>
+        <xref linkend="ST_CoverageUnion" />,
+        <xref linkend="ST_CoverageSimplify" />
+      </para>
+    </refsection>
+
+  </refentry>
+
+
+  <refentry id="ST_CoverageSimplify">
+    <refnamediv>
+      <refname>ST_CoverageSimplify</refname>
+
+      <refpurpose>Window function that simplifies the edges of an implicit coverage formed by a set of polygons, applying the Visvalingam–Whyatt algorithm to the edges.</refpurpose>
+    </refnamediv>
+
+    <refsynopsisdiv>
+      <funcsynopsis>
+        <funcprototype>
+          <funcdef>geometry <function>ST_CoverageSimplify</function></funcdef>
+          <paramdef><type>geometry winset </type>
+            <parameter>geom</parameter></paramdef>
+          <paramdef><type>float8 </type>
+            <parameter>tolerance</parameter></paramdef>
+          <paramdef choice="opt"><type>integer </type>
```

Comment:
Why not a boolean?

Reply:
The idea is to leave the option open for a "Simplify Outer Only" choice as well.  But not sure that will ever eventualize, and I agree that a `boolean` is simpler.  

We could make this a boolean meaning "simplify inner edges only" with default `false`.  If needed another boolean option meaning "simplify outer edges only" with default `false`.

Reply:
Yeah, the flag makes sense for GEOS where we can't have function overloads, but PostGIS doesn't have that limitation.